app/services/current-user.ts

```diff
@@ -31,7 +31,7 @@ function hashCode(str: string): number {
     return str
         .split('')
         // eslint-disable-next-line no-bitwise
-        .reduce((acc, _, i) => ((acc << 5) - acc) + str.charCodeAt(i), 0); // tslint:disable-line no-bitwise
+        .reduce((acc, _, i) => ((acc << 5) - acc) + str.charCodeAt(i), 0); // eslint-disable-line  no-bitwise
```

Comment:
nit: `no-bitwise` is already disabled from the previous line (nice that we no longer need it twice!)